### PR TITLE
project-schema.json: add social.inIndigenousLand and .landCompensatio…

### LINKED ISSUE #407: land compensation budget: specify budgte set aside for land compensation claims
# Background

This issue relates to the following CoST IDS elements proposed in the CoST IDS/OC4IDS review:

<details>

<summary> Land compensation budget </summary>

## Land compensation budget

**Module:** Social
**Indicator:** Land issues

### Disclosure format

> Disclose budget allocated to fund land compensation (E.g. [value]). 

### OC4IDS mapping

> Project Level: Set `.social.landCompensationBudget` to the amount and currency of the budget allocated for land compensation. 

</details>

# Proposal

Add `Value` object to new `social` object (see #403) titled `landCompensationBudget`.

Add the following fields and objects:

<div dir="ltr" style="margin-left:0pt;" align="left" id="docs-internal-guid-f7d4070d-7fff-6723-31e7-b59de769f878">

Path | Title | Description | Type (format) | Codelist
-- | -- | -- | -- | --
social.landCompensationBudget | Land compensation budget | The budget allocated in the contract for compensation of land. | `Value` object | -

</div>

# Example

```json

{ 
 "social": { 
   "landCompensationBudget": { 
 	"amount": 1000000, 
 	"currency": "USD" 
   } 
 } 
} 

```
cc @evelyndinora

### LINKED ISSUE #408: identify projects overlap with Indigenous land
# Background

This issue relates to the following CoST IDS elements proposed in the CoST IDS/OC4IDS review:

<details>

<summary> Indigenous land </summary>

## Indigenous land

**Module:** Social
**Indicator:** Indigenous populations

### Disclosure format

> Identify whether the project is located or cut through indigenous land. Use the information at the LandMark – Global Platform of Indigenous and Community Lands on both databases Indigenous Lands Acknowledged by Government and Not Acknowledged by Government (customary tenure or with formal land claim submitted) to disclose the information. 

### OC4IDS mapping

> Project level:  If the project is located or cut through indigenous land: 1. Set `.social.indigenousLand` to `true` 2 Add a `Location` object to the `.locations` array, set its `.id` incrementally and set its description to “Indigenous land: <Name> (<Category>)” substituting <Name>  and <Category> for the name and land category from the Landmark database. If the project is not located or cut through indigenous land, set `.social.indigenousLand` to `false`. 

</details>

# Proposal

Add a boolean `inIndigenousLand` field to the new `social` object (see #403).

Add the following fields and objects:

<div dir="ltr" style="margin-left:0pt;" align="left" id="docs-internal-guid-83213628-7fff-4c47-b46d-453ca5a86454">

Path | Title | Description | Type (format) | Codelist
-- | -- | -- | -- | --
social.inIndigenousLand | In Indigenous land | Whether the project is located in or cuts through Indigenous land. | boolean | -

</div>

# Example

```json

{ 
 "social": { 
   "inIndigenousLand": true 
 }, 
 "locations": [ 
   { 
 	"id": "1", 
 	"description": "Indigenous land: Section 1 Block III Rowallan Survey District (Maori Freehold Land)" 
   } 
 ] 
} 

```

# Sources

* [LandMark: The Global Platform of Indigenous and Community Lands](https://www.landmarkmap.org/data/)

cc @evelyndinora

### PR DESCRIPTION
**Pull request by odscjen**
…nBudget

**Related issues**

closes #407 
closes #408 

**Description**

both small changes so made sense to combine them into a single PR. Removed mention of "in the contract" from the description of `landCompensationBudget` as realised this is likely to be paid by the project, not the supplier and the contract wasn't mentioned by CoST in the original discussion. Also if it is in the contract then this field should be in `contractingProcess.summary.social` rather than just in `social`

**Merge checklist**

- [x] [Log your changes](https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#logging-changes)

If there are changes to `project-schema.json` or `project-package-schema.json`:

- [x] Update the examples:
  - [ ] `docs/examples/example.json`
  - [ ] `docs/examples/blank.json`
- [x] Run `./manage.py pre-commit` to update `docs/_static/i8n.csv`

If you added a new definition to the schema, run `./manage.py pre-commit`.

If you added a new codelist:

  - [ ] Add an entry to `docs/reference/codelists.md`

### CODE REVIEW

File: docs/examples/blank.json [hunk 1]
```diff
@@ -524,7 +524,12 @@
           "jobTitle": "string"
         }
       }
-    ]
+    ],
+    "landCompensationBudget": {
+      "amount": "integer",
```
Comment (duncandewhurst) [inline docs/examples/blank.json]:
```suggestion
      "amount": "number",
```

File: mapping/sustainability.yaml [hunk 2]
```diff
@@ -1245,20 +1245,20 @@
     title: Indigenous land
     module: Social
     indicator: Indigenous populations
-    disclosure format: Identify whether the project is located or cut through indigenous land. Use the information at the LandMark - Global Platform of Indigenous and Community Lands on both databases Indigenous Lands Acknowledged by Government and Not Acknowledged by Government (customary tenure or with formal land claim submitted) to disclose the information.
+    disclosure format: Identify whether the project is located or cut through indigenous land. Use the information at the [LandMark - Global Platform of Indigenous and Community Lands](https://www.landmarkmap.org/) on both databases Indigenous Lands Acknowledged by Government and Not Acknowledged by Government (customary tenure or with formal land claim submitted) to disclose the information.
     mapping: |-
         Project level:
 
-        If the project is located or cut through indigenous land:
+        If the project is located in or cuts through Indigenous land:
```
Comment (duncandewhurst) [inline mapping/sustainability.yaml]:
```suggestion
        If the project is located in or cuts through indigenous land:
```

File: mapping/sustainability.yaml [hunk 3]
```diff
@@ -1245,20 +1245,20 @@
...
 
-        1. Set `.social.indigenousLand` to `true`
+        1. Set `.social.inIndigenousLand` to `true`
            2 Add a `Location` object to the `.locations` array, set its `.id` incrementally and set its description to "Indigenous land: <Name> (<Category>)" substituting <Name>  and <Category> for the name and land category from the Landmark database.
```
Comment (duncandewhurst) [inline mapping/sustainability.yaml]:
```suggestion
        2. Add a `Location` object to the `.locations` array, set its `.id` incrementally and set its description to "Indigenous land: <Name> (<Category>)" substituting <Name>  and <Category> for the name and land category from the Landmark database.
```

File: schema/project-level/project-schema.json [hunk 4]
```diff
@@ -2468,6 +2468,16 @@
           "items": {
             "$ref": "#/definitions/Meeting"
           }
+        },
+        "landCompensationBudget": {
+          "title": "Land compensation budget",
+          "description": "The budget allocated for compensation of land.",
```
Comment (duncandewhurst) [inline schema/project-level/project-schema.json]: 'Compensation of land' reads oddly.

```suggestion
          "description": "The budget allocated for land compensation.",
```

File: schema/project-level/project-schema.json [hunk 5]
```diff
@@ -2468,6 +2468,16 @@
...
+          "$ref": "#/definitions/Value"
+        },
+        "inIndigenousLand": {
+          "title": "In Indigenous land",
+          "description": "Whether the project is located in or cuts through Indigenous land.",
```
Comment (duncandewhurst) [inline schema/project-level/project-schema.json]: Not sure why indigenous is capitalised.

```suggestion
          "title": "In indigenous land",
          "description": "Whether the project is located in or cuts through indigenous land.",
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

**Comment by duncandewhurst** [comment]
I agree that land compensation is not the responsibility of suppliers so I think this belongs at the project level.

Jen is on leave today so, in the interest of expediency, I've committed my own suggestions and run `update-sustainability-docs`.

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**